src/player_compevents.c

```diff
@@ -116,7 +116,7 @@ Comp_Event_Func computer_event_func_list[] = {
   NULL,
 };
 
-//PowerKind pwkind; char gaction; char require_owned_ground; int repeat_num; int pwlevel; int amount_able;
+//PowerKind pwkind; char gaction; char require_owned_ground; int repeat_num; CrtrExpLevel pwlevel; int amount_able;
```

Comment:
Is this right? Powers can accept a level less than creatures right?

Reply:
I mean, powers take levels 0~8 but creatures have levels 0~9.

Reply:
It means that powers are something else and the values should not be confused. It's what bullfrog did to get the level 10 spell bug.